lib/config/types.go

```diff
@@ -48,11 +49,17 @@ func (p Project) SourceRepo() Repo {
 // A RepositoryType describes a type of revision control system which manages target revisions of deployment.
 type RepositoryType string
 
+type HostType string
```

Comment:
The HostType could be k8s or a normal instance. If it is k8s, it will retrieve the commit hash from the RC definition.

http://goship-k8staging-01.gengo.com/

Note only validator is working now.